pylint/checkers/classes.py

```diff
@@ -133,6 +133,26 @@ def _positional_parameters(method):
         positional = positional[1:]
     return positional
 
+def _has_different_parameters_default_value(original, overridden):
```

Comment:
Just a small nitpick, I'd really love more insightful (and less abbreviated) names used in local names of this functions:
- `orig_name` should really be pluralized (it's collection of names) - maybe `original_param_names`?
- `original_default` and `overridden_default` may be slightly easier to comprehend than current abbreviated forms.